content/news/052/index.md

```diff
@@ -78,6 +78,25 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library Updates
 
+### [egui_ratatui][egui_ratatui]
+
+![egui_ratatui running in bevy](egui_ratatui.jpg)
+
+[egui_ratatui][egui_ratatui] by [gold-silver-copper][gold] 
+- is an [egui][egui] widget that is also a [ratatui][ratatui] backend.
+- It allows you to create Terminal User Interfaces inside Egui.
+- Can be used with bevy, macroquad, eframe, pixels, etc
+- It is also WASM compatible.
+- The current release is the product of months of iteration, and is now "stable".
+- It is currently being used for a game and at a startup, with no issues.
```

Comment:
`a game and at a startup` -> I'm curious about that, which game, which startup ?

Reply:
@Vrixyz  The game is a roguelike in the interslavic language. The startup is GraphicsGrid Interactive Education, of which I am a member, we create educational software in various disciplines including language learning, geography, and geology. Our CEO is currently a FAANG engineer with experience in multiple other startups.

Thank you for your curiosity.